src/pipx/venv.py

```diff
@@ -156,24 +156,28 @@ def main_package_name(self) -> str:
         else:
             return self.pipx_metadata.main_package.package
 
-    def create_venv(self, venv_args: List[str], pip_args: List[str]) -> None:
+    def create_venv(
+        self, venv_args: List[str], pip_args: List[str], override_shared: bool = False
+    ) -> None:
```

Comment:
Can we have a docstring describing `override_shared`?

Reply:
Added.